### DISCUSSION TITLE
api/LogOut get called on external link

### DISCUSSION THREAD

**Discussion by seanaguinaga** [gladly-team/next-firebase-auth#423] [Q&A]
logout gets called when leaving the app

login is not called when hitting the browser back button, not even with refreshes - the tab must be closed and re-opened

Is this intentional?

Am I doing something wrong?

**Comment by seanaguinaga** [comment]
The token does not work server side until the tab is closed and reopened, it does not work server side on refresh

**Comment by kmjennison** [comment]
Please provide a reproduction and more detail so others can potentially help.

I'd recommend setting up the [example](https://github.com/gladly-team/next-firebase-auth/tree/main/example) app to see if the problem happens there.
